### PR TITLE
Merge master 02/10/2017

### PR DESCRIPTION
**Pull request by Pandrex247**

### CODE REVIEW

**Comment by Pandrex247** [comment]
Jenkins, quick test Payara 5 please

File: appserver/packager/glassfish-web/pom.xml [hunk 1]
```diff
@@ -234,7 +234,15 @@ Portions Copyright [2016, 2017] [Payara Foundation and/or its affiliates]
...
-        </dependency>       
+        </dependency>
+        <dependency>
+            <groupId>javax.security.enterprise</groupId>
+            <artifactId>javax.security.enterprise-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.soteria</groupId>
+            <artifactId>javax.security.enterprise</artifactId>
+        </dependency>
```
Comment (arjantijms) [inline appserver/packager/glassfish-web/pom.xml]: This would not have needed to be done, Soteria already in 5 ;)

File: appserver/pom.xml [hunk 2]
```diff
@@ -699,6 +699,16 @@
                 <artifactId>javax.security.auth.message-api</artifactId>
                 <version>${javax.security.auth.message-api.version}</version>
             </dependency>
+                   <dependency>
+                <groupId>javax.security.enterprise</groupId>
```
Comment (arjantijms) [inline appserver/pom.xml]: This would not have needed to be done, Soteria already in 5 ;)

**Comment by Pandrex247** [comment]
Updated with changes

**Comment by Pandrex247** [comment]
Jenkins, quick test Payara-5 please

**Comment by Pandrex247** [comment]
Jenkins, quick test Payara 5 please

**Pull request merged**